src/components/Header/Header.jsx

```diff
@@ -1,19 +1,29 @@
-import { Link } from 'react-router-dom';
+import { Link, useNavigate, useLocation } from 'react-router-dom';
 import styled from 'styled-components';
 import { Button } from '../Button';
 
 const Header = () => {
+  const navigate = useNavigate();
+  const location = useLocation(); // 현재 경로를 가져옴
+
+  const GoLogin = () => {
+    if (location.pathname !== '/login') {
+      // 현재 경로가 /Login이 아닌 경우에만 이동
+      navigate('/login');
+    }
+  };
+
   return (
     <StHeader>
       <StContents>
-        <StLink href="/">
+        <StLink to="/">
           ✍🏻{' '}
           <StHeading>
             <StStrong>Blood</StStrong>folio
           </StHeading>
         </StLink>
-        <Button variant="secondary" href="/profile" rounded>
-          내 프로필
+        <Button onClick={GoLogin} variant="secondary" rounded>
```

Comment:
```suggestion
        <Button variant="secondary" href="/login" rounded>
```
페이지만 이동하는 것이라면 onClick을 지양해주세요!